src/mediawiki/pages/music/music.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  MediaWikiBuilder,
+  MediaWikiTemplate,
+  MediaWikiText,
+} from "@osrs-wiki/mediawiki-builder";
+
+import InfoboxMusicTemplate from "@/mediawiki/templates/InfoboxMusic";
+import { DBRow } from "@/utils/cache2";
+
+export const musicPageBuilder = (dbRow: DBRow) => {
```

Comment:
Instead of passing a `DBRow` to `musicPageBuilder` and `InfoboxMusicTemplate`, create a `MusicTrack` type in `src/types/music.ts` to represent the values of the music dbrow's.